types/budgets.d.ts

```diff
@@ -0,0 +1,36 @@
+/**
+ * @license Copyright 2019 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+
+declare global {
+    module LH {
+        module Budgets {
+            // More info: https://github.com/GoogleChrome/lighthouse/issues/6053#issuecomment-428385930
```

Comment:
can you do this in the jsdoc style `/** */` so they'll show up on hover (at least in vscode, maybe other editors...).

Also can you add short docs for all the interfaces/properties below?